project1/css/index.css

```diff
@@ -62,3 +62,128 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+
+.container {
+	width: 1200px;
+	border: 2px solid red;
+	margin: 0 auto;
+  }
+  
+  a {
+	color: grey;
+	text-decoration: none;
+	font-size: 15px;
+  }
+  
+  header nav ul li {
+	display: inline-block;
+	margin-left: 75px;
+	margin-top: 40px;
+	color: inherent;
+  }
+  
+  header img {
+	display: block;
+	width: 40%;
+	border: 1px solid green;
+	height: 100px;
+	width: 270px;
+	position: absolute;
+	right: 140px;
+	top: 30px;
+  }
+  
```

Comment:
By adding the position as absolute to your image some interesting things start to happen with scaling.

![absolute positioning error](https://user-images.githubusercontent.com/12449668/37049357-e60d80e4-212d-11e8-9216-153f18b7a7ee.png)

If you decide to polish this up be sure to remove borders